doc/API/centreon-api-v2.yaml

```diff
@@ -2513,8 +2513,8 @@ components:
           example: "3/3 (H)"
         last_check:
           type: string
-          format: date-time
           description: "Date of the last check (ISO8601)"
+          example: "1h 45m"
```

Comment:
if the date is ISO8601 this example can't be accurate

Reply:
the description is changed